production/kustomization.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+- https://github.com/redhat-appstudio/jvm-build-service/deploy/crds/base?ref=7debedb49322ecd1b5b4009b71c2623412ffd30e
+- https://github.com/redhat-appstudio/jvm-build-service/deploy/base?ref=7debedb49322ecd1b5b4009b71c2623412ffd30e
+- https://github.com/redhat-appstudio/jvm-build-service/deploy/operator/config?ref=7debedb49322ecd1b5b4009b71c2623412ffd30e
+- https://github.com/redhat-appstudio/jvm-build-service/deploy/operator/base?ref=7debedb49322ecd1b5b4009b71c2623412ffd30e
+# TODO Do we need the cluster monitoring from operator/overlays/dev-template? Could reference the files directly.
+
+# NB: From operator/overlays/dev-template
+images:
+  - name: hacbs-jvm-operator
+    newName: quay.io/redhat-appstudio/hacbs-jvm-controller
+    newTag: dev
+
+# NB: From overlays/dev-template
+patches:
+# TODO: Remove first (quaytoken) once https://github.com/redhat-appstudio/jvm-build-service/pull/915 is merged
+- patch: |-
+    - op: replace
+      path: /spec/template/spec/volumes/0/secret/optional
+      value: true
+  target:
+    kind: Deployment
+    name: hacbs-jvm-operator
+- patch: |-
+    - op: replace
+      path: /spec/steps/2/image
+      value: quay.io/redhat-appstudio/hacbs-jvm-build-request-processor:dev
+  target:
+    kind: Task
+    name: maven
+- patch: |-
+    - op: replace
+      path: /spec/steps/2/image
+      value: quay.io/redhat-appstudio/hacbs-jvm-build-request-processor:dev
+  target:
+    kind: Task
+    name: gradle
```

Comment:
I _looks_ like https://github.com/redhat-appstudio/jvm-build-service/blob/main/deploy/overlays/dev-template/kustomization.yaml#L17 is missing the Gradle one?